webapp/channels/src/components/channel_header_dropdown/menu_items/view_pinned_posts/view_pinned_posts.tsx

```diff
@@ -19,33 +19,32 @@ type Props = {
     };
 }
 
-export default class ViewPinnedPosts extends React.PureComponent<Props> {
-    private handleClick = (e: React.MouseEvent) => {
+const ViewPinnedPosts = ({
+    channel,
+    hasPinnedPosts,
+    actions: {
+        closeRightHandSide,
+        showPinnedPosts,
+    },
+    show,
+}: Props) => {
+    const handleClick = (e: React.MouseEvent) => {
```

Comment:
You are passing this to a child component, so better make it a callback (useCallback) to make sure you don't send a new function on every render.

Reply:
Done. 🙌